crates/oxc_linter/src/snapshots/anchor_has_content.snap

```diff
@@ -0,0 +1,48 @@
+---
+source: crates/oxc_linter/src/tester.rs
+assertion_line: 119
+expression: anchor_has_content
+---
+  ⚠ eslint-plugin-jsx-a11y(anchor-has-content): Missing accessible content when using `a` elements.
+   ╭─[anchor_has_content.tsx:1:1]
+ 1 │ <a />
+   · ─────
```

Comment:
In oxc we try to minimize the pointing span that shows the error, in this case it would be helpful to just point to `a` here.